geneaprove.py

```diff
@@ -12,9 +12,9 @@
     
     db = settings.DATABASES['default']['NAME']
     if not os.path.isfile(db):
-        print "======================="
-        print "Creating a new database %s" % db
-        print "======================="
+        print(f"=======================")
```

Comment:
doesn't need to be an f-string

Reply:
Mistake on my part. I meant to do:

```
        print("=======================")
        print(f"Creating a new database {db}")
        print("=======================")
```

And just did the top and bottom as well without thinking. Corrected.